components/tools/FiltersDropdown.tsx

```diff
@@ -0,0 +1,57 @@
+import { twMerge } from 'tailwind-merge';
+
+interface DataListType {
+  name: string;
+  color: string;
+  borderColor: string;
+};
+
+interface FiltersDropdownProps {
+  dataList?: DataListType[];
+  checkedOptions?: string[];
+  setCheckedOptions: React.Dispatch<React.SetStateAction<string[]>>;
+  className?: string;
+};
+
+/**
+ * @description This component displays Filter Dropdown Component.
+ *
+ * @param {DataListType[]} props.dataList - List of filter options.
+ * @param {string[]} props.checkedOptions - List of options that are currently checked.
+ * @param {React.Dispatch<React.SetStateAction<string[]>>} props.setCheckedOptions - Function to set check state of options.
+ * @param {string} props.className - Additional CSS classes for the component.
+ */
+export default function FiltersDropdown({
+  dataList = [],
+  checkedOptions = [],
+  setCheckedOptions,
+  className = ''
+}: FiltersDropdownProps) {
+  const handleClickOption = (event: React.MouseEvent, option: string) => {
```

Comment:
Should I also pass `checkedOptions` and `setCheckedOptions` to this function similar to what passed to `handleClickOption` function in the `components/tools/FilterDisplay.tsx` or should I also remove these arguments from that function too?